RetrospectiveExtension.Frontend/utilities/external/telemetryClient.ts

```diff
@@ -7,11 +7,26 @@ const appInsights = new ApplicationInsights({
   config: {
     instrumentationKey: environment.AppInsightsInstrumentKey,
     extensions: [reactPlugin],
+    loggingLevelConsole:2,
+    loggingLevelTelemetry:2,
     extensionConfig: {
       [reactPlugin.identifier]: { history: {} }
     }
   }
 });
 appInsights.loadAppInsights();
 
+// The new log error function logs the data to console as usual,
+// but also sends error messages to Application insights as well.
+const updatedConsoleError=(function(oldErrorFunction){
+  return {
+      error: function(message?: any, ...optionalParams: any[]){
```

Comment:
Please add `// eslint-disable-next-line @typescript-eslint/no-explicit-any` line above to avoid warning issues with lint.

Reply:
good point